passenger-rails/2.3/nginx.conf.erb

```diff
@@ -56,6 +56,18 @@ http {
   # Performance Helps
   sendfile on;
 
+  server {
```

Comment:
This now makes this specific dockerfile YEI specific. Will this not fit in the `include /usr/src/nginx/conf.d/*.conf` files and be done in the YEI-Rails dockerfile?

Reply:
Is there a file in the YEI-Rails repo? I looked in there first, but didn't see it.

Reply:
Currently we haven't added anything to that directory in YEI-Rails. The way that this docker file was build was to be flexible and expanded. If you add files to the right directory they'll be automatically picked up as part of your app's nginx config (We even get crazy and let you do erb in the config files - see: https://github.com/youearnedit/rundmc/pull/160)

So what you'd do is:
1. Create the nginx conf file you want
2. Add it to the right place in the app's Dockerfile: `COPY config/redirects.conf /usr/src/nginx/conf.d/redirects.conf`
3. When you start up the app it'll load up nginx with the config (see: https://github.com/youearnedit/dockerfiles/blob/master/passenger-rails/2.3/nginx.conf.erb#L54)

If you've got any question let me know.